service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -626,17 +626,15 @@ public ResponseEntity<ApiCloneWorkspaceResult> cloneWorkspace(
     //  this part, we do the retrieval and permission check using the workspace spend profile.
     Optional<SpendProfileId> spendProfileIdOptional =
         Optional.ofNullable(body.getSpendProfile()).map(SpendProfileId::new);
-    SpendProfileId spendProfileId =
-        spendProfileIdOptional.orElse(
-            sourceWorkspace
-                .getSpendProfileId()
-                .orElseThrow(
-                    () ->
-                        MissingSpendProfileException.forWorkspace(sourceWorkspace.workspaceId())));
 
-    SpendProfile spendProfile =
-        spendProfileService.authorizeLinking(
-            spendProfileId, features.isBpmGcpEnabled(), petRequest);
+    if (spendProfileIdOptional.isEmpty()) {
+      spendProfileIdOptional = sourceWorkspace.getSpendProfileId();
```

Comment:
Just curious -- under what scenarios would a workspace not have a spend profile?  I see that this field is an Optional.